test/integration/minishift.go

```diff
@@ -190,6 +190,43 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 	return nil
 }
 
+func (m *Minishift) setImageCaching(operation string) error {
+	enabled := "true"
+	if operation == "disabled" {
+		enabled = "false"
+	}
+
+	return m.executingMinishiftCommand(fmt.Sprintf("config set image-caching %s", enabled))
+}
+
+func (m *Minishift) imageExportShouldComplete(noOfImages int) error {
+	// poll till the output of the `minishift image list` shows three OpenShift cached images
```

Comment:
`three` should be changed to `number of images` or `noOfImages`.

Reply:
@agajdosi Yes, forgot to update comment :) when I added argument.